src/System.Runtime.WindowsRuntime/src/System/Threading/Tasks/AsyncInfoToTaskBridge.dotnet53.cs

```diff
@@ -120,6 +120,11 @@ private void Complete(IAsyncInfo asyncInfo, Func<IAsyncInfo, TResult> getResults
             if (asyncInfo == null)
                 throw new ArgumentNullException("asyncInfo");
             Contract.EndContractBlock();
+            
+            if (System.Threading.Tasks.Task.s_asyncDebuggingEnabled)
+			{
+				System.Threading.Tasks.Task.RemoveFromActiveTasks(base.Task.Id);
+			}
```

Comment:
Nit: indentation is off here and below... looks like tabs are being used instead of spaces?